jekyll/_data/ja/sidenav.yml

```diff
@@ -368,6 +369,15 @@ en:
             link: 2.0/server-3-operator-backup-and-restore
           - 
           - 
+      - 
```

Comment:
Ah sorry for confusion, noboru-san said that this file may not be referenced. But both of us didn't have the courage to delete this file...

Reply:
sure, i understand.
however, if that is the case, i would think the updates here are not correct and not needed in any case? then
hence, i am not sure why there is an update here, and for wrong information i believe 🤔 (since there is no server 2.38)